shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -140,9 +140,9 @@
   NSObject<FlutterPlatformViewFactory>* factory = factories_[viewType].get();
   if (factory == nil) {
     result([FlutterError errorWithCode:@"unregistered_view_type"
-                               message:[NSString stringWithFormat:@"A UIKitView widget is trying to create a PlatformView with an unregistered type: < %@ >", viewType]
+                               message:[NSString stringWithFormat:@"A UIKitView widget is trying to create a PlatformView with an unregistered type: < %@ >", args[@"viewType"]]
```

Comment:
Can you pull `args[@"viewType"]` into a local above `std::string viewType([args[@"viewType"] UTF8String]);` and use it here to reduce the complexity of this line?

Reply:
@jmagman `std::string viewType([args[@"viewType"] UTF8String]);` is already defined on line 132. The problem that I noticed by using only `viewType` as variable was that the build failed with "error: cannot pass non-trivial object of type 'std::string'". So I guess it needs an NSString and not an std::String

Reply:
Actually I think it should work if I use %s instead of %@

Reply:
Don't use a `%s` c string like that, there are encoding implications https://stackoverflow.com/a/60217425

I was suggesting
```objc
NSString* viewTypeString = [args[@"viewType"];
std::string viewType(viewTypeString.UTF8String);
...
              message:[NSString stringWithFormat:@"A UIKitView widget is trying to create a "
                                                 @"PlatformView with an unregistered type: < %@ >",
                                                 viewTypeString]
```

Reply:
@jmagman I added the NSString variable there. I was thinking that maybe I have to make it work using the existing variable.